pkg/apis/bundle/v1alpha1/helpers_test.go

```diff
@@ -219,7 +219,7 @@ func TestParseURL(t *testing.T) {
 				}
```

Comment:
Idk why Github doesn't allow me to comment on it, but rather than using `t.Errorf` / `return`, now about just `t.Fatalf`?

Reply:
Done